src/drivers/pwm_input/pwm_input.cpp

```diff
@@ -294,6 +293,12 @@ PWMIN::~PWMIN()
  * to be started in init scripts when the user may be using the input
  * pin as PWM output
  */
+
+static void _freeze_test()
```

Comment:
@dagar - RE: stack sizes. For the long term we should look at having 2 settings that adjust the values, one based on arch (cpu type) the other tool chain.